zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/CMSFeatureFlags.java

```diff
@@ -106,6 +118,9 @@ public boolean isJwtSessionsEnabled() {
 
     public boolean isKafkaEnabled() { return isKafkaEnabled; }
 
+    public boolean isStaticFilesPublishingEnabled() {return isStaticFilesPublishingEnabled;}
```

Comment:
```suggestion
    public boolean isStaticFilesPublishingEnabled() {
           return isStaticFilesPublishingEnabled;
    }
```
Small change but this makes it more readable. I would suggest to do the same for the `isKafkaEnabled()` so that then everything is consistent in terms of formatting